mesonbuild/environment.py

```diff
@@ -96,17 +96,43 @@ def detect_gcovr(min_version: str = '3.3', log: bool = False):
         return gcovr_exe, found
     return None, None
 
-def detect_llvm_cov():
+def detect_llvm_cov(suffix: T.Optional[str]):
```

Comment:
I would add a default initializer of None here. Then it saves you from changing the test code

Reply:
Ah, good shout! 👍🏼